src/01-general/users.apib

```diff
@@ -51,6 +51,10 @@ Get a list of all users.
         + filter (object, optional)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `John` (string, optional) - Filters on first name, last name and email
+            + status: `all` (enum[string], optional) - Filters on status
+                + Members
+                    + all - Filters on all active and deactivated users
```

Comment:
PP: I believe `all` does not make much sense because IMO filtering all is the same as not using this filter at all.

Reply:
Actually, we decided that that would be a breaking thing.

Currently, we do not return deactivated users, and since it doesn't make sense to always return deactivated users we decided to do it like this.

Reply:
Indeed, good point! Anyway, I would remove the `all` and would allow an array of status to be filtered on.

Reply:
the `all` case is actually a good reason do make the filter an array...

Reply:
Yes I agree, that argument convinced me to change the implementation.